### ISSUE TITLE
Barrierefreiheit: Kontrast Überschrift Termine-Seite

### ISSUE THREAD

**Issue by albig** [verdigado/sunflower#979]
pa11y und andere Tools beschweren sich auf der Termine-Seite über den Kontrast der Überschrift:

```
Results for URL: https://wordpress71.gcms.verdigado.net/termine/

 • Error: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 2.24:1. Recommendation:  change text colour to #70a30a.
   ├── WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
   ├── #primary > header > h1
   └── <h1 class="page-title is-visible"> Termine </h1>

1 Errors
```
